packages/design-system-react-native/src/components/Box/Box.tsx

```diff
@@ -75,10 +75,10 @@ export const Box = ({
     ${borderWidth !== undefined ? TWCLASSMAP_BOX_BORDER_WIDTH[borderWidth] : ''}
     ${borderColor ?? ''}
     ${backgroundColor ?? ''}
-    ${twClassName}`.trim();
+    ${twClassName}`;
 
   return (
-    <View style={[tw`${twContainerClassNames}`, style]} {...props}>
+    <View style={[twContainerClassNames, style]} {...props}>
```

Comment:
We should try to avoid this way of defining tailwind classnames in react native because unless the classnames appear in `tw`` tailwind intellisense and the eslint tailwind plugin won't work.  We should likely refactor all of our react-native components to fix this